Tabloid/client/src/components/posts/PostDetails.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Card, CardBody } from "reactstrap";
+import { useState, useEffect } from "react";
+import { getPostById } from "../../modules/postManager";
+import { useParams } from "react-router-dom";
+
+const PostDetails = () => {
+  const [post, setPosts] = useState({});
```

Comment:
The naming convention here between the state variable and it's setter is inconsistent (singular v plural).. is this intentional? If so, what is the reason?

Reply:
I have an unhealthy relationship with my 's' key. I tend to use it when I don't need to and forget when I do.

Reply:
I have corrected that naming convention and gave some others more intention names.